pkg/spec/parse.go

```diff
@@ -138,22 +138,25 @@ func validateIOpsDevice(val string) (*throttleDevice, error) { //nolint
 }
 
 // getLoggingOpts splits the path= and tag= options provided to --log-opt.
-func getLoggingOpts(opts []string) (string, string) {
+func getLoggingOpts(opts []string) (string, string, string) {
```

Comment:
Since this list could grow over time, could you return a map

map[string]string
func getLoggingOpts(opts []string) map[string]string {
}

Then the code above gan just reference it via  name
```
if logmap["path"] != "" {
     		options = append(options, libpod.WithLogPath( logmap["path"]))
}
```

Reply:
This way if sometime in the future we add "compression" or something else, we don't need to change the function interface.

Reply:
Thanks for this valuable suggestion. Incorporated and reflected this change in my new commit and updated this branch.

Reply:
I need help as make validation step as facing the below issue:

diff -u ./pkg/spec/createconfig.go.orig ./pkg/spec/createconfig.go
--- ./pkg/spec/createconfig.go.orig	2020-12-10 18:56:04.945670929 +0530
+++ ./pkg/spec/createconfig.go	2020-12-10 18:56:04.946754272 +0530
@@ -314,9 +314,9 @@
 	logPath := logMap["path"]
 	logTag := logMap["tag"]
 	logSizeMax := logMap["logsizemax"]
-	
+
 	intlogsizemax, err := strconv.ParseInt(logSizeMax, 10, 64)
-	
+
 	if err != nil {
 		return nil, err
 	}
make: *** [Makefile:173: gofmt] Error 1

Any pointers @rhatdan  ? thanks